src/api/googlePlacesApiLoader.js

```diff
@@ -0,0 +1,7 @@
+import { useJsApiLoader } from "@react-google-maps/api";
+import React from "react";
+
+export const GooglePlacesApiLoader = (data) => {
```

Comment:
We seperated the functionality of the useJsApiLoader which is used to call the google places autocomplete api